File: app/src/main/java/carbonylgroup/com/schoolpower/classes/DetailsTransition.java
/**
 * Copyright (C) 2016 Gustav Wang
 */

package carbonylgroup.com.schoolpower.classes;

import android.transition.ChangeBounds;
import android.transition.ChangeImageTransform;
import android.transition.ChangeTransform;
import android.transition.TransitionSet;


public class DetailsTransition extends TransitionSet {

    public DetailsTransition() {
        setOrdering(ORDERING_TOGETHER);
        addTransition(new ChangeBounds()).
                addTransition(new ChangeTransform());
                addTransition(new ChangeImageTransform());
    }

}

File: app/src/main/java/com/carbonylgroup/schoolpower/classes/Transition/DetailsTransition.kt
/**
 * Copyright (C) 2016 Gustav Wang
 */

package com.carbonylgroup.schoolpower.classes.Transition

import android.transition.ChangeBounds
import android.transition.ChangeImageTransform
import android.transition.ChangeTransform
import android.transition.TransitionSet


class DetailsTransition : TransitionSet() {
    init {
        ordering = TransitionSet.ORDERING_TOGETHER
        addTransition(ChangeBounds()).addTransition(ChangeTransform())
        addTransition(ChangeImageTransform())
    }

}
